app/views/customers/_form.html.erb

```diff
@@ -1,9 +1,10 @@
-<%= simple_form_for(@customer) do |f| %>
+<%= simple_form_for(@presenter.resource) do |f| %>
   <%= f.error_notification %>
 
   <div class="form-inputs">
-    <%= f.input :name %>
-    <%= f.input :email %>
+    <% @presenter.attribute_names.each do |attr_name| %>
```

Comment:
Hm... `attribute_names` is a different interface than the `ShowPresenter` uses...